src/sentry/models/organization.py

```diff
@@ -161,6 +161,16 @@ def get_default_owner(self):
             sentry_orgmember_set__organization=self,
         )[0]
 
+    def has_single_owner(self):
+        from sentry.models import OrganizationMember, OrganizationMemberType
+        count = OrganizationMember.objects.filter(
+            organization=self,
+            type=OrganizationMemberType.OWNER,
+            has_global_access=True,
+            user__isnull=False,
+        ).count()
```

Comment:
I guess continuing reading, this is being used already inside of an O(n) loop, so probably worth it.